great-idea/css/index.css

```diff
@@ -69,76 +69,71 @@ h1, h2, h3, h4, h5 {
 	border: 1px solid black;
 	width: 888px;
 	margin: 0px auto;
-	vertical-align: center;
 	display: flex;
 	flex-direction: column;
-	align: center;
-
+	
 }
 
 	.container-header {
-		border: 4px solid red;
-		/* width: 880px; */
-		height: 250px;
-		margin: 0px 10px 0px 10px;
+		/* border: 4px solid red; */
+		height: 280px;
 		display: flex;
 		flex-direction: column;
 		justify:center;
-		/* z-index: 1; */
-		/* justify-content: space-around; */
 	}
 		
 		.box-nav {
-			border: 1px solid red;
-			text-decoration: none;
-			width: 95%;
-			height: 20px; 
+			/* border: 3px solid yellow; */
+			/* text-decoration: none; */
+			width: 100%;
+			height: 50px; 
 			display: flex;
-			justify-content: flex-start;
+			justify-content: space-between;
 
 		}
 		
-		/* .box-logo {
-			border: 1px solid red;
-			display: flex;
-			width: 49%;
-			justify-content: flex-end;
-		} */
-
 		.box-services-header {
-			border: 10px solid green;
-			/* display: flex; */
-			
-			
+			/* border: 3px solid green; */
+			width: 880px;
+			display: flex;
 		}
+			.services-header-img {
+				width: 100%;
+			}
 	
 	.container-services {
 		/* border: 3px solid purple; */
-		margin: 10px;
+		height: 120px;
+		margin: 5px;
+		border-bottom: 2px solid black;
 	}
 		.box-services {
-			border: 1px solid red;
+			/* border: 1px solid red; */
 			flex-direction: column;
 			display: flex;
 			justify-content: space-around;
-			border-bottom: 2px solid black;
+			
 		}
 
 	.container-main {
-		border: 3px solid blue;
+		/* border: 3px solid blue; */
 		height: 600px;
 		display: flex;
```

Comment:
Nice job getting the flex to work. A few things though. The default flex-direction is row, so it isn't necessary to add that property. Justify and align aren't css properties by themselves. Be sure to check that the styles you are adding to your css are doing something. If they aren't doing anything delete them. Are you using your inspector tools in your browser?